specs/rollup-node.md

```diff
@@ -251,6 +253,46 @@ However, they can still be challenged by a fault proof until the end of the faul
 
 > **TODO** the spec doesn't encode the notion of fault proof yet, revisit this (and include links) when it does
 
+## From L2 Block to L2 Output Root
+
+After processing a block the resulting outputs will need to be synchronized with L1 for trustless execution of
+L2-to-L1 messaging, such as withdrawals. Outputs are hashed in a tree structured form for minimal proof cost to any
+piece of data. This merkle-structure is defined with [SSZ], a type system for merkleization and serialization, used in
+L1 (beacon-chain). However, we replace `sha256` with `keccak256` to save gas costs in the EVM.
```

Comment:
s/tree structured/tree-structured

Reply:
s/for minimal proof cost to any piece of data/which minimizes the cost of proving any piece of data captured by the outputs/

We probably need a glossary entry for the concept of proof, I'll make an issue for it.

Reply:
This is the first time the word "output" appears in this document, and so we need to define what it means.
Since the `L2Output` structure is defined right below, I think just linking to the `Block` entry of the glossary (which defines block outputs) should be enough.

Reply:
add a TODO for adding links to "withdrawals" once we introduce them in the spec

Reply:
I am not sure about the exact output wording / glossary usage, feel free to change it